src/azul/deployment.py

```diff
@@ -91,18 +91,15 @@ def es_endpoint(self) -> Optional[Netloc]:
 
     @property
     def es_instance_count(self) -> Optional[int]:
```

Comment:
Now this seems to be identical to the `config` property with the same name. Does it make sense to remove this proxy in favour of the `config` attribute?